Musakai.Blazor/Services/ApiServices/ApiContactService.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Musakai.Interfaces.ApiServices;
+using Musakai.Shared.Api.Contact.Requests;
+using System.Net.Http.Json;
+
+namespace Musakai.Blazor.Services.ApiServices;
+
+/// <summary>
+/// Service for sending contact messages to the Photographer
+/// </summary>
+public class ApiContactService : IApiContactService
+{
+    private readonly HttpClient _httpClient;
+    private Uri _baseEndpoint = new Uri("https://api.danielklement.net/v1/contact");
```

Comment:
Be careful with hardcoding the URLs. I'd recommend some centralized point where you can configure this in first place. Or fetching these thing dynamically, because it will use some kind of bindings (address to IP + port).

Reply:
Hardcoded just for now, I wanted to make a structure for future work, where I'll have probably EnvVariables, or something like that.

Reply:
That's something you might want to do next. Or as I said, make it dynamic.

Reply:
Changed logic of processing endpoints, also added versioning.